Run task at 2:15am

proteusvacuum opened the pull request:
@nickpell 

This was getting run at midnight UTC (5:30am IST), which was cutting it close to for completion before the work day started in India.

proteusvacuum committed:
Run task at 2:15am

emord commented:
I'm not sure if https://github.com/dimagi/commcarehq-ansible/pull/921 was ever deployed. I meant to do it, but I wasn't ansible setup on eniksay.

proteusvacuum commented:
@emord ah yeah, forgot about that. I'll deploy localsettings once I get confirmation from the channel that the other changes are safe.

proteusvacuum committed:
Timezone will be Asia/Kolkata

https://github.com/dimagi/commcarehq-ansible/commit/88d0e647788bcccbebe12a382a2580e032632ca5
